### PR TITLE
Add contributing.md

### LINKED ISSUE #171: Add CONTRIBUTING.md
*It looks like you're contributing to KICL. Would you like help?*

### PR DESCRIPTION
**Pull request by mbax**
Resolves #171

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+[Briefly search the issues](https://github.com/KittehOrg/KittehIRCClientLib/issues)
+to see if there is an existing issue covering this bug.
+Respond as you feel appropriate, if there is one.
+
+### 2. Create a new issue
+
+[Create an issue](https://github.com/KittehOrg/KittehIRCClientLib/issues/new)!
+Be sure to include as much of the following as you can:
+
+ * Version of the client library.
```
Comment (Zarthus) [inline CONTRIBUTING.md]: Can be psuedo enforced by adding a [`.github/ISSUE_TEMPLATE/Bug_Report.md`](https://github.com/jkcclemens/paste/tree/master/.github/ISSUE_TEMPLATE)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * Steps to reproduce (Example code, preferably!).
+ * Stacktrace observed.
+ * Version of the client library where it was working before, if you knew of one.
```
Comment (bendem) [inline CONTRIBUTING.md]: I'd ask about the target operating system too (linux/windows/macos/android).

Comment (lol768) [reply CONTRIBUTING.md]: + the JVM version

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[Create an issue](https://github.com/KittehOrg/KittehIRCClientLib/issues/new)!
+Be as thorough as you can with your suggestion.
+Explain why you feel it should be added to the library.
+
+## I want to contribute code!
+
+Cool! Consider [creating an issue](https://github.com/KittehOrg/KittehIRCClientLib/issues/new)
+first for an idea that isn't already accepted as a feature to be added or
+[talk about it on IRC](https://webchat.esper.net/?nick=kicl_...&channels=%23kitteh.org&prompt=1)
```
[the remark this answers is not in the document]
Comment (mbax) [reply CONTRIBUTING.md]: This is just copypasted from my old code from prior to the webchat change. Thanks for letting me know :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (mbax) [reply CONTRIBUTING.md]: Thanks! I'll do that.

**Pull request merged**